pkg/command/plugin.go

```diff
@@ -457,7 +462,8 @@ func newCleanPluginCmd() *cobra.Command {
 func newSyncPluginCmd() *cobra.Command {
 	var syncCmd = &cobra.Command{
 		Use:   "sync",
-		Short: "Sync the plugins",
+		Short: "Installs all recommended plugins from the active contexts",
+		Long:  `Installs all recommended plugins from the active contexts. Plugins installed with this command will only be available until the context for which this plugins is active and might not be visible when the context is not active`,
```

Comment:
Can you break this into multiple lines?  Here the forward quotes are good to allow you to do that.

```suggestion
		Long:  `Installs all recommended plugins from the active contexts.
Plugins installed with this command will only be available while the context remains active.`,
```